ql/src/java/org/apache/hadoop/hive/ql/stats/BasicStatsNoJobTask.java

```diff
@@ -307,7 +330,16 @@ public void run() {
         console.printInfo("[Warning] could not update stats for " + partish.getSimpleName() + ".", "Failed with exception " + e.getMessage() + "\n" + StringUtils.stringifyException(e));
       }
     }
+  }
 
+  public static Consumer<String> smallFilesConsoleAndWarn(Logger log) {
```

Comment:
if this method is just for a simple message, we can just move it to the caller, e.g,
```
.ifPresent(s -> {
      SessionState ss = SessionState.get();
      if (ss != null && ss.getConsole() != null) {
        ss.getConsole().printInfo(msg);
      } else {
         LOG.warn(msg);
      }
 }
```

Reply:
Acknowledged.